MinnPost/object-sync-for-salesforce
# Fieldmap screen design updates

## What does this PR do?

1. Closes #328.
2. Closes #319.
3. Closes #276.

## How do I test this PR?

1. Make sure the upgrade process runs smoothly without PHP errors.
2. Make sure the push and pull still works as expected even if no changes are made to existing setup.
3. Make a new fieldmap. Make sure it pushes and pulls successfully.
4. Edit an existing fieldmap. Make sure it pushes and pulls successfully.
